src/NuGet.Clients/NuGet.PackageManagement.UI/PackageItemLoader.cs

```diff
@@ -140,6 +146,15 @@ private async ValueTask<INuGetSearchService> GetSearchServiceAsync(CancellationT
             return searchService;
         }
 
+        private async ValueTask<INuGetRemoteFileService> GetRemoteFileServiceAsync(CancellationToken cancellationToken)
+        {
+#pragma warning disable ISB001 // Dispose of proxies
```

Comment:
I thought we were doing file-based suppressions, not inline ones

Reply:
good question...searching the codebase, i find many pragmas...and searching for ISB001, i only find inline uses.
Scanning https://github.com/NuGet/NuGet.Client/blob/dev/docs/coding-guidelines.md, i didn't see a mention one way or the other. gonna keep this one the same for now, but would love to understand right direction and place in docs to note this.